doc/source/whatsnew/v0.18.2.txt

```diff
@@ -139,3 +144,10 @@ Bug Fixes
 - Bug in ``NaT`` - ``Period`` raises ``AttributeError`` (:issue:`13071`)
 - Bug in ``Period`` addition raises ``TypeError`` if ``Period`` is on right hand side (:issue:`13069`)
 - Bug in ``pd.set_eng_float_format()`` that would prevent NaN's from formatting (:issue:`11981`)
+
+
+
```

Comment:
put these higher up in the bug fix list (almost randomly place them) (so doesn't mess up conflicts with others)